waltz-test/src/main/python/waltz_ducktape/tests/validation/scalability_test.py

```diff
@@ -28,6 +31,15 @@ def test_scale_up_replica(self, num_active_partitions, txn_per_client, num_clien
         self.run_produce_consume_validate(lambda: self.scale_up_replica(src_node_idx, added_node_idx, num_active_partitions,
                                                                         txn_per_client, num_clients, interval, timeout))
 
+    @cluster(cluster_spec=MIN_CLUSTER_SPEC)
+    @parametrize(num_active_partitions=4, txn_per_client=500, num_clients=1, interval=100, timeout=240, num_added_nodes=1, delay_before_torture=20)
```

Comment:
I think we can just limit `test_scale_up_server_nodes` to add a new server node after the initial Waltz Cluster is created. So, `num_added_nodes` is not needed. Because, testing to scale up the server node by 1 would test/verify the same behavior that we would if we scale up the server nodes by more than 1 I feel.

And instead of adding `delay_before_torture` parameter, we can determine when to add a new server based on once at least one transaction is written to Waltz.